sync/class.jetpack-sync-settings.php

```diff
@@ -58,9 +60,16 @@ static function get_setting( $setting ) {
 			$value = intval( $value );
 		}
 
-		// specifically for the post_types blacklist, we want to include the hardcoded settings
-		if ( $setting === 'post_types_blacklist' ) {
-			$value = array_unique( array_merge( $value, Jetpack_Sync_Defaults::$blacklisted_post_types ) );
+		switch( $setting ) {
```

Comment:
we can probably do this more efficiently by detecting if the stored value is an array, e.g.

```php
switch( $setting ) {
  case 'post_types_blacklist':
    $default_array_value = Jetpack_Sync_Defaults::$blacklisted_post_types;
    $merge_arrays = true;
  //... same for others
}

if ( $merge_arrays && is_array( $value ) ) {
  $value = array_unique( array_merge( $value, $default_array_value ) );
}

```